lib/utils/logUtil.ts

```diff
@@ -2,7 +2,9 @@ import winston, { format, Logger } from "winston";
 
 const PROCESSOR_NAME = "ldes-client";
 
-const consoleTransport = new winston.transports.Console();
+const consoleTransport = new winston.transports.Console({
+    stderrLevels: ["error", "warn"]
```

Comment:
What's the thought behind this change? Should we update our other processors accordingly?

Reply:
My main reason for doing this is that I want to enable the CLI interface to be piped into other tools or write its output to disk using standard unix commands. For example:
```bash
$ ldes-client https://myldes.org | my_other_tool
$ ldes-client https://myldes.org > ./myFile.nt
```
And for this, the `stdout` needs to be clean, which is why I configure winston to send `error` and `warn` messages to stderr instead. That way you can still see them in the console but they won't pollute the client's output.

Now that I think about it, probably `verbose` and `debug` should also be in there. What do you think @smessie?

Reply:
Yes, while reading your comment, it felt like none of the logging should go to stdout.
So `verbose` and `debug`, but also `info` should maybe go to `stderr`. Then we keep `stdout` clean for processors that print output to `stdout`.
This should also be something to change over time in the other processors then.

From https://github.com/winstonjs/winston/tree/master?tab=readme-ov-file#logging-levels the full list should be:
`emerg`, `alert`, `crit`, `error`, `warning`, `warn`, `notice`, `info`, `http`, `verbose`, `debug`, `silly`

Reply:
:+1:

Reply:
Agreed. In https://github.com/rdf-connect/ldes-client/pull/94/commits/79cfc00c3b4cc386f7e58228980b2962c14aed35 I made all winston logs to be written to `stderr`. Now also the `--quiet` option will only shutdown writing the members to `stdout`. In this way if one wants to just check the logs it can be done with:
```bash
$ DEBUG=* ldes-client http://myldes.org -q
```
Which is equivalent to:
```bash
$ DEBUG=* ldes-client http://myldes.org > /dev/null
```